devimint/src/federation.rs

```diff
@@ -30,16 +32,59 @@ impl Federation {
     pub async fn new(
         process_mgr: &ProcessManager,
         bitcoind: Bitcoind,
-        vars: BTreeMap<usize, vars::Fedimintd>,
+        servers: usize,
+        // vars: BTreeMap<usize, vars::Fedimintd>,
     ) -> Result<Self> {
         let mut members = BTreeMap::new();
-        for (peer, var) in &vars {
+        let mut vars = BTreeMap::new();
+
+        // FIXME: this is a hack to pass fed.server_gen_params in
+        // local_config_gen_params call below
+        let fed = FedimintBuilder::new()?.with_default_modules();
+        let peers: Vec<_> = (0..servers).map(|id| PeerId::from(id as u16)).collect();
+        let params: HashMap<PeerId, ConfigGenParams> =
+            local_config_gen_params(&peers, BASE_PORT, fed.server_gen_params.clone())?;
+
+        let mut admin_clients: BTreeMap<PeerId, WsAdminClient> = BTreeMap::new();
+        for (peer, peer_params) in &params {
+            // FIXME: we should do this inside
```

Comment:
@justinmoon should we also drop the fixme?

Reply:
not sure what i meant here, just delete comment